src/utils/factory.js

```diff
@@ -0,0 +1,9 @@
+export const generateTaskObject = (title) => {
+  return {
+    id: Date.now().toString(),
+    title: title,
```

Comment:
// see https://eslint.org/docs/latest/rules/object-shorthand

```suggestion
    title,
```